src/app/shared/constants/options/conditions.ts

```diff
@@ -61,7 +61,7 @@ export const CONDITION_DISTANCE_TO_CV3: Option[] = [
   { value: 1,	name: 'Distance must be higher than ConditionValue2' },
   { value: 2,	name: 'Distance must be lower than ConditionValue2' },
   { value: 3,	name: 'distance must be equal to or higher than ConditionValue2' },
-  { value: 3,	name: 'distance must be equal to or lower than ConditionValue2' },
+  { value: 4,	name: 'distance must be equal to or lower than ConditionValue2' },
```

Comment:
these changes aren't related with the PR's title

Reply:
+ this is fixed on Master